tests/target/single_line_if/true_100.rs

```diff
@@ -0,0 +1,57 @@
+// rustfmt-single_line_simple_if: true
+// rustfmt-unstable_features: true
+// rustfmt-single_line_if_else_max_width: 100
+
+fn main() {
+    let a = if 1 > 2 { unreachable!() } else { 10 };
+
+    let a = if x {
+        1
+    } else if y {
+        2
+    } else {
+        3
+    };
+
+    if true { continue }
+
+    if true { continue }
+
+    if width == 50_characters_or_shorter { continue }
```

Comment:
I'm pretty sure that this line `    if width == 50_characters_or_shorter { continue }` has a total width of 53, which is a bit misleading in my opinion. I'd want to see cases where the entire `if` including the conditional expression and the `block` come to a width of `width_setting` - 1, `width_setting`, `width_setting` + 1. so if the `width_setting=50`, I'd want to see test cases where the width is 49, 50, 51, and 51+. If we introduce this, then we want to make sure that we wrap appropriately, and having tests focused on testing the bounds should help.